rust/hyperlane-base/src/types/gcs_storage.rs

```diff
@@ -7,6 +7,7 @@ use std::fmt;
 use ya_gcp::{storage::StorageClient, AuthFlow, ClientBuilder, ClientBuilderConfig};
 
 const LATEST_INDEX_KEY: &str = "gcsLatestIndexKey";
```

Comment:
not within the scope of this PR, but I didn't realize the keys for the GCS checkpoint syncer are different than s3, I'm a little worried this'll make debugging GCS stuff unnecessarily annoying. Because GCS isn't technically live (https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/4069) @daniel-savu I wonder if we should also change this as part of 4069?